docs/yamlpages/protocols.rst

```diff
@@ -114,14 +114,20 @@ Automatic Path Determination
          alchemical_path: auto
 
 
-YANK can automatically determine the parameters for ``lambda_X`` for you. YANK will distribute states between the fully
+YANK can automatically determine the sequence of values for the ``lambda_X`` values using an algorithm we called
+"thermodynamic trailblazing". YANK will distribute states between the fully
```

Comment:
We should also document the user-specifiable settings that can affect how trailblazing is conducted, like the number of equilibration steps, the number of samples/steps per energy recorded, the step size, etc.

Reply:
I agree. I was thinking of adding docs for the options when I'll expose them (see previous comment).